### DISCUSSION TITLE
Renaming the domain URL

### DISCUSSION THREAD

**Discussion by Venugopal1998** [community/community#52151] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hi Team,

I want to change the existing GitHub.com Domain URL to New GitHub.com domain URL. Could you please assist me on this particular thing.

**Example:**
I have **company1.github.com** domain URL. So, I want to change/rename it to new domain URL as **company2.githib.com**. Could it possible to rename it with new domain name?

Thanks,
Venugopal Reddy.

**Comment by meppu** [comment]
You can create a new organization and move your website repository to that organization name.

Lets say you want `new-company-llc.github.io`. In this case, you should create an organization with name `new-company-llc` and fork your current pages repository to organization. [Creating a New Organization](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/creating-a-new-organization-from-scratch)

You can also rename your current name. If you want to switch `old-company-llc` to `new-company-llc` simple rename your name as `new-company-llc` and update your pages repository name. [Changing Your GitHub Username](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username)

**Comment by mehereesh** [comment]
✅ Option 1: Create a New Organization
Create a GitHub organization with the desired name (e.g., new-company-llc).

Fork your existing old-company-llc.github.io repository to the new organization.

Your new site will be at: https://new-company-llc.github.io

🔗 [Creating a New Organization](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/creating-a-new-organization-from-scratch)

✅ Option 2: Rename Your Username
Rename your current GitHub username from old-company-llc to new-company-llc.

Update the repository name to new-company-llc.github.io.

GitHub will automatically redirect your old URLs.

🔗 [Changing Your GitHub Username](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username)
